src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -126,6 +126,12 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv)
             }
         }
 
+        $returnNull = false;
+        if ('' === $prefix) {
```

Comment:
`''` becomes a magic value that acts as the `string` prefix except for null where we return null